cypress/integration/executions_spec.js

```diff
@@ -69,5 +69,59 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show a single execution', () => {
+      cy.contains('nav li a', 'Executions').as('executions');
+      cy.get('@executions').should('have.attr', 'href', '#/executions');
+      cy.get('@executions').click();
+
+      cy.url().should('include', 'executions');
+      cy.contains('.heading--xlarge', 'Executions');
+      const executionName = '50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf:50eaad71-bba8-4376-83d7-bb9cc1309b92';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestGranuleStateMachine-MOyI0myKEXzf';
+      cy.get('table tbody tr td[class=table__main-asset]').within(() => {
+        cy.get(`a[title=${executionName}]`).click({force: true});
+      });
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow');
+
+      cy.get('.table--wrapper')
+        .within(() => {
+          let i;
+          for (i = 26; i < 32; i++) {
+            let idMatch = `"id": ${i},`;
+            let previousIdMatch = `"previousEventId": ${i - 1}`;
+
+            cy.get(`tr[data-value=${i}]`).children('td').as('items');
+            cy.get('@items').eq(0).should('have.text', i.toString());
+            cy.get('@items').eq(2).invoke('text').should('match', /\d{2}:\d{2}:\d{2} \d{2}\/\d{2}\/\d{2}$/);
+            cy.get('@items').eq(3).contains('More Details').click();
+            cy.get('@items').eq(3).contains(idMatch).contains(previousIdMatch);
+            cy.get('@items').eq(3).contains('Less Details').click();
+          }
+        });
+
+      cy.get('.status--process')
+        .within(() => {
+          cy.contains('Execution Status:').next().should('have.text', 'Succeeded');
+          cy.contains('Execution Arn:').next().should('have.text', executionArn);
+          cy.contains('State Machine Arn:').next().should('have.text', stateMachine);
+          cy.contains('Started:').next().should('have.text', '20:05:10 11/12/18');
+          cy.contains('Ended:').next().should('have.text', '20:05:31 11/12/18');
+          cy.contains('Logs:').next()
+            .within(() => {
+              cy.get('a').should('have.attr', 'href', `#/executions/execution/${executionName}/logs`).click();
+            });
+        });
+
+      cy.contains('.heading--large', `Logs for Execution ${executionName}`);
```

Comment:
It feels like this should be a separate test that starts with a `cy.visit` for the URL to the logs page. If nothing else, it breaks up this test which is already quite large